On linux I am getting another error when entering the local path:
```
>>> Mount storage to /home/jovyan/work? (y/n): y
>>> Enter local path: /home/pablo/test_storage
Traceback (most recent call last):
  File "/home/pablo/python/python3.5.2/bin/resen", line 9, in <module>
    load_entry_point('resen==19.1.0', 'console_scripts', 'resen')()
  File "/home/pablo/python/python3.5.2/lib/python3.5/site-packages/resen/resencmd.py", line 446, in main
    ResenCmd(res).cmdloop(intro)
  File "/usr/lib/python3.5/cmd.py", line 138, in cmdloop
    stop = self.onecmd(line)
  File "/usr/lib/python3.5/cmd.py", line 217, in onecmd
    return func(arg)
  File "/home/pablo/python/python3.5.2/lib/python3.5/site-packages/resen/resencmd.py", line 69, in do_create_bucket
    local_path = self.get_valid_local_path('>>> Enter local path: ')
  File "/home/pablo/python/python3.5.2/lib/python3.5/site-packages/resen/resencmd.py", line 388, in get_valid_local_path
    if os.path.isdir(path):
  File "/usr/lib/python3.5/genericpath.py", line 42, in isdir
    st = os.stat(s)
TypeError: argument should be string, bytes or integer, not PurePosixPath
```

Linux: 4.4.0-141-generic #167-Ubuntu SMP Wed Dec 5 10:40:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
python version: 3.5.2
docker version: Docker version 18.09.1, build 4c52b90

Now, if I try using `stop_jupyter bucket_name` in the resen tool  in a bucket without storage(because it didn't work) that I let jupyter start at the end of the guided create_bucket command, the resen tool also hangs (as it did on my mac)

--- a/resen/resencmd.py
+++ b/resen/resencmd.py
@@ -389,7 +389,7 @@ stop_jupyter bucket_name : Stop jupyter on bucket bucket_name."""
         while True:
             path = input(msg)
             path = pathlib.PurePosixPath(path)
-            if os.path.isdir(path):
+            if os.path.isdir(str(path)):
                 return str(path)
             else:
                 print('Cannot find local path entered.')

--- a/resen/resencmd.py
+++ b/resen/resencmd.py
@@ -174,7 +174,7 @@ remove_bucket bucket_name : Remove bucket named bucket_name."""
 
     def do_start_jupyter(self,args):
         """Usage:
->>> start_jupyter bucket_name
+>>> start_jupyter bucket_name : Start jupyter on bucket bucket_name
         """
         inputs,num_inputs = self.parse_args(args)
 

--- a/resen/resencmd.py
+++ b/resen/resencmd.py
@@ -188,6 +188,10 @@ remove_bucket bucket_name : Remove bucket named bucket_name."""
         # get bucket name from input
         bucket_name = inputs[0]
 
+        if not bucket_name in self.program.bucket_manager.bucket_names:
+            print("ERROR: Bucket with name: %s does not exist!" % bucket_name)
+            return False
+
         # get bucket infomrmation (ports and status)
         # This stuff may be better suited to exist in some kind of "status query" inside of Resen.py
         ind = self.program.bucket_manager.bucket_names.index(bucket_name)

